analyses/dndscv-analysis/.gitignore

```diff
@@ -0,0 +1 @@
+input/hg38.fa.gz.1
```

Comment:
I think this file is generated by accident, such as download fail and re-download the file again. If everything works fine, we do not need to include this file in gitignore